src/core/api/index.js

```diff
@@ -12,9 +12,12 @@ import { convertFiltersToQueryParams } from 'core/searchUtils';
 
 const API_BASE = `${config.get('apiHost')}${config.get('apiPath')}`;
 
-export const addon = new Schema('addons', { idAttribute: 'slug' });
-export const category = new Schema('categories', { idAttribute: 'slug' });
-export const user = new Schema('users', { idAttribute: 'username' });
+export const addon = new normalizrSchema.Entity(
```

Comment:
This is a lot longer than I'd like. Maybe save `Entity` off into a const and use that?